packages/databases-collections-list/src/namespace-card.tsx

```diff
@@ -262,7 +300,22 @@ export const NamespaceItemCard: React.FunctionComponent<
       {...cardProps}
     >
       <CardTitleGroup>
-        <CardName>{name}</CardName>
+        <CardName isNonExistent={isNonExistent}>{name}</CardName>
+
+        {isNonExistent && (
+          <Tooltip
+            align="bottom"
+            justify="start"
+            trigger={
+              <div className={tooltipTriggerStyles}>
+                <Icon glyph={'InfoWithCircle'} />
+              </div>
+            }
+          >
+            You have privileges to this namespace, but it is not in your list of
+            current namespaces
```

Comment:
Have we ran this text by product? On first read I found this text a bit confusing. Maybe instead of `it is not in your list of current namespaces` we mention that the namespaces wasn't returned when we listed collections/databases?

Reply:
Changed in [523154c](https://github.com/mongodb-js/compass/pull/6664/commits/523154cc4beb118e87fd50de5bed1ce6fa9b2e0d). Ran it through Betsy!